blockmanager.go

```diff
@@ -1981,6 +2027,12 @@ func checkCFCheckptSanity(cp map[string][]*chainhash.Hash,
 func (b *blockManager) blockHandler() {
 	defer b.wg.Done()
 
+	// Attempt to sideLoad headers. If sideLoading is not enabled the function
+	// returns quickly.
+	b.sideLoadHeaders()
+	// TODO(Maureen): update candidate peers while sideloading so that we
```

Comment:
We should do this even earlier, we want to side load all we have before trying to sync to the network.

Reply:
So this actually sideloads before syncing to the network as it is here. Are you saying before the block manager even starts?

Reply:
Yes before it even starts. This is basically a pre-processing step. This way we can further decouple it from all the main logic.